src/components/Camera/index.js

```diff
@@ -133,13 +147,25 @@ export default class Camera extends React.Component<CameraType> {
     this.capture.stop()
   }
 
-  render = () => (
-    <CameraPure {...{
-      ...this.props,
-      faceCaptureClick: this.capture.once,
-      webcamRef: (c) => { this.webcam = c },
-      onFallbackClick: () => {this.stopCamera},
-    }}
-    />
-  )
+  setPermissionsPrimerSeen = () => {
+    this.setState({ hasSeenPermissionsPrimer: true })
+  }
+
+  render = () => {
+    const { hasSeenPermissionsPrimer, hasWebcamAccess } = this.state;
+    return (
+      (hasWebcamAccess || hasSeenPermissionsPrimer) ?
```

Comment:
I think we always to see the permission primer in case the permission has not been granted. this means that is not the case, right?

Reply:
Yes, the user will always see the permissions primer if permissions haven't been granted.

The `hasSeenPermissionsPrimer` flag is only to stop showing the primer once the user clicked `'Enable webcam'` and show the camera capture component.

Reply:
I see